clientconn.go

```diff
@@ -1681,14 +1680,14 @@ func (cc *ClientConn) connectionError() error {
 //
 // Doesn't grab cc.mu as this method is expected to be called only at Dial time.
 func (cc *ClientConn) parseTargetAndFindResolver() error {
-	channelz.Infof(logger, cc.channelz, "original dial target is: %q", cc.target)
+	logger.Infof("original dial target is: %q", cc.target)
```

Comment:
The problem with this simple conversion is that the channelz version would prefix with info about the channel.

Let's clean things up here.

- This log message is not helpful, or if it is, it should be moved into `NewClient` and left as channelz-scoped.